notebooks/model_conversion-pca.ipynb

```diff
@@ -0,0 +1,427 @@
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "source": [
+    "# Load breast cancer wisconsin dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "f619a434",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-04-03T08:30:21.028485Z",
+     "iopub.status.busy": "2023-04-03T08:30:21.028183Z",
+     "iopub.status.idle": "2023-04-03T08:30:21.044606Z",
+     "shell.execute_reply": "2023-04-03T08:30:21.043725Z"
+    },
+    "papermill": {
+     "duration": 0.026204,
+     "end_time": "2023-04-03T08:30:21.046731",
+     "exception": false,
+     "start_time": "2023-04-03T08:30:21.020527",
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "random_state=0\n",
+    "X, y = datasets.load_breast_cancer(return_X_y=True)\n",
+    "X_train, X_test, y_train, y_test = sklearn.model_selection.train_test_split(X, y, \n",
+    "                                      test_size=0.3, random_state=random_state)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "af30c5ac",
+   "metadata": {
+    "papermill": {
+     "duration": 0.006551,
+     "end_time": "2023-04-03T08:30:21.060014",
+     "exception": false,
+     "start_time": "2023-04-03T08:30:21.053463",
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "source": [
+    "# Initialize and fit the models to training set"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "b8734b45",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-04-03T08:30:21.074460Z",
+     "iopub.status.busy": "2023-04-03T08:30:21.074185Z",
+     "iopub.status.idle": "2023-04-03T08:30:21.079004Z",
+     "shell.execute_reply": "2023-04-03T08:30:21.077863Z"
+    },
+    "papermill": {
+     "duration": 0.01476,
+     "end_time": "2023-04-03T08:30:21.081377",
+     "exception": false,
+     "start_time": "2023-04-03T08:30:21.066617",
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "n_components = 15\n",
+    "sklearn_models = [\n",
+    "    PCA(n_components=n_components),\n",
+    "    KernelPCA(n_components=n_components),\n",
+    "    TruncatedSVD(n_components=n_components)\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "7e9d1846",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-04-03T08:30:21.095788Z",
+     "iopub.status.busy": "2023-04-03T08:30:21.095482Z",
+     "iopub.status.idle": "2023-04-03T08:30:21.147753Z",
+     "shell.execute_reply": "2023-04-03T08:30:21.146433Z"
+    },
+    "papermill": {
+     "duration": 0.063221,
+     "end_time": "2023-04-03T08:30:21.151195",
+     "exception": false,
+     "start_time": "2023-04-03T08:30:21.087974",
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "for model in sklearn_models:\n",
+    "    model.fit(X_train)"
```

Comment:
Right now this is throwing a not-converged warning for the SVM classifier. While training convergence doesn't actually matter for the purposes of this notebook, it might be good to add a brief text comment here acknowledging that.